terraform/project.tf

```diff
@@ -0,0 +1,31 @@
+variable "project_name" {}
+variable "billing_account" {}
```

Comment:
It's also best practice to put all variables into a separate `vars.tf` file -- see https://github.com/cisagov/crossfeed/blob/master/infrastructure/vars.tf for an example.